.github/ISSUE_TEMPLATE/config.yml

```
@@ -0,0 +1,9 @@
+blank_issues_enabled: true
+contact_links:
+  - name: Stack Overflow
+    url: https://stackoverflow.com/questions/tagged/vowpalwabbit
+about: Please ask questions or search for solutions here
```

This is invalid yaml

```suggestion
   about: Please ask questions or search for solutions here
```